examples/api/.env.example

```diff
@@ -3,7 +3,7 @@ MONGO_URL=mongodb://localhost:27017/wepublish
 WEBSITE_URL=http://localhost:5000
 
 MEDIA_SERVER_URL=http://localhost:3004
-MEDIA_SERVER_TOKEN=secret
+MEDIA_SERVER_TOKEN=tPcvBRM2B3uSulyxjXm2ciqH5f1vQy2VDAsf
```

Comment:
`secret` is the right media server token here. In the `examples/media/.env` you'll find the following environment variable `TOKEN=secret`. We could choose a more complex one but it's for local development purposes and therefore okay. If you want to deploy a productive we.publish instance you have to setup a more complex password.